cline_docs/settings.md

```diff
@@ -2,31 +2,31 @@
 
 1. Add the setting to schema definitions:
 
-    - Add the item to `globalSettingsSchema` in `schemas/index.ts`
-    - Add the item to `globalSettingsRecord` in `schemas/index.ts`
+    - Add the item to `globalSettingsSchema` in `src/schemas/index.ts`
+    - Add the item to `globalSettingsRecord` in `src/schemas/index.ts`
     - Example: `terminalCommandDelay: z.number().optional(),`
 
 2. Add the setting to type definitions:
 
-    - Add the item to `exports/types.ts`
-    - Add the item to `exports/roo-code.d.ts`
-    - Add the setting to `shared/ExtensionMessage.ts`
-    - Add the setting to the WebviewMessage type in `shared/WebviewMessage.ts`
+    - Add the item to `src/exports/types.ts`
+    - Add the item to `src/exports/roo-code.d.ts`
```

Comment:
No idea if this is an issue, but it's at the very least amusing that the file is referring to a roo-derived file-name in a cline-derived path.